services/app-web/src/pages/LinkYourRates/LinkYourRates.test.tsx

```diff
@@ -46,80 +52,322 @@ describe('LinkYourRates', () => {
-            >
-                <form>
-                    <LinkYourRates
-                        fieldNamePrefix="rateForms.1"
-                        index={1}
-                        autofill={jest.fn()}
-                    />
-                </form>
-            </Formik>,
+            <Routes>
+                <Route
+                    path={RoutesRecord.SUBMISSIONS_RATE_DETAILS}
+                    element={<RateDetailsV2 type="MULTI" />}
+                />
+            </Routes>,
             {
                 apolloProvider: {
                     mocks: [
-                        fetchCurrentUserMock({
-                            statusCode: 200,
-                            user: mockValidStateUser(),
+                        fetchCurrentUserMock({ statusCode: 200 }),
+                        fetchDraftRateMockSuccess({ id: rateID }),
+                        fetchContractMockSuccess({
+                            contract: {
+                                id: 'test-abc-123',
+                            },
                         }),
-                        indexRatesMockSuccess(),
                     ],
                 },
+                routerProvider: {
+                    route: `/submissions/test-abc-123/edit/rate-details`,
+                },
+                featureFlags: {
+                    'link-rates': true,
+                    'rate-edit-unlock': false,
+                },
             }
         )
 
-        const noRadioButton = await screen.getByLabelText(
+        //Making sure page loads
+        await screen.findByText('Rate Details')
+        expect(
+            screen.getByText(
+                'Was this rate certification included with another submission?'
+            )
+        ).toBeInTheDocument()
+
+        //Click the no button and assert the dropdown does not appear
+        const noRadioButton = screen.getByLabelText(
             'No, this rate certification was not included with any other submissions'
         )
         await user.click(noRadioButton)
-
         await waitFor(() => {
             expect(
                 screen.queryByText('Which rate certification was it?')
             ).not.toBeInTheDocument()
         })
     })
-    //eslint-disable-next-line
-    it.skip('displays dropdown menu if yes is selected', async () => {
+
+    it('displays dropdown menu if yes is selected', async () => {
+        const rateID = 'test-abc-123'
         const { user } = renderWithProviders(
-            <Formik
-                initialValues={{ ratePreviouslySubmitted: '' }}
-                onSubmit={(values) => console.info('submitted', values)}
-            >
-                <form>
-                    <LinkYourRates
-                        fieldNamePrefix="rateForms.1"
-                        index={1}
-                        autofill={jest.fn()}
-                    />
-                </form>
-            </Formik>,
+            <Routes>
+                <Route
```

Comment:
As written, these are now RateDetailsV2 page level integration tests. They are not component level tests for `<LinkYourRates />` anymore.  Instead, logic here hinges `RateDetailsV2` continuing to be set up a certain way= and they will fail if that parent page file changed.  

I'm fine with that! However, please just move them up into `RateDetailsV2.test.tsx` then - put them together in their own nested describe block, e.g. `can link existing rate`